testdata/baselines/reference/submoduleAccepted/conformance/checkJsdocSatisfiesTag2.types.diff

```diff
@@ -7,8 +7,9 @@
 ->p : { isEven: (n: number) => boolean; isOdd: (n: number) => boolean; }
 ->({    isEven: n => n % 2 === 0,    isOdd: n => n % 2 === 1}) : { isEven: (n: number) => boolean; isOdd: (n: number) => boolean; }
 ->{    isEven: n => n % 2 === 0,    isOdd: n => n % 2 === 1} : { isEven: (n: number) => boolean; isOdd: (n: number) => boolean; }
-+>p : { isEven: (n: any) => boolean; isOdd: (n: any) => boolean; }
-+>({    isEven: n => n % 2 === 0,    isOdd: n => n % 2 === 1}) : { isEven: (n: any) => boolean; isOdd: (n: any) => boolean; }
++>p : any
```

Comment:
this is now `any` because Corsa no longer supports index signature syntax like `Object.<string, T>`